### DISCUSSION TITLE
如何给REFRESH命令的执行权限

### DISCUSSION THREAD

**Discussion by Python-IBM** [apache/doris#25622] [A - General / Q&A]
普通账号使用REFRESH命令时提示没有权限，如何通过命令给新建的普通账号执行REFRESH命令。

**Comment by LemonLiTree** [comment]
ALTER_PRIV, CREATE_PRIV, DROP_PRIV

**Reply by Python-IBM** [reply]
这些都有给，但普通账号还是没有权限使用REFRESH命令

**Reply by zddr** [reply]
List your SQL here?

**Reply by Python-IBM** [reply]
1、GRANT SELECT_PRIV,LOAD_PRIV,ALTER_PRIV,CREATE_PRIV,DROP_PRIV ON <catalog_name>.<db_name>.* TO 'develop'@'%';
2、REFRESH CATALOG <catalog_name>;

**Reply by ixzc** [reply]
You may have misunderstood the issues related to authorization. Your command should be: GRANT SELECT_PRIV,LOAD_PRIV,ALTER_PRIV,CREATE_PRIV,DROP_PRIV ON <catalog_name>. * . * TO 'develop'@'%';
https://doris.apache.org/zh-CN/docs/sql-manual/sql-reference/Account-Management-Statements/GRANT

**Comment by Python-IBM** [comment]
> List your SQL here?



> List your SQL here?
